module/enrichers.mjs

```diff
@@ -106,12 +106,56 @@ function parseConfig(match, { multiple=false }={}) {
  *   <i class="fa-solid fa-dice-d20" inert></i> +5
  * </a>
  * ```
+ *
+ * @example Create an attack link using a specific attack mode:
+ * ```[[/attack +5]]``` or ```[[/attack formula=5 attackMode=thrown]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="attack" data-formula="+5" data-attack-mode="thrown">
+ *   <i class="fa-solid fa-dice-d20" inert></i> +5
+ * </a>
+ * ```
+ *
+ * @example Link an enricher to an attack activity, either explicitly or automatically
+ * ```[[/attack activity=RLQlsLo5InKHZadn]]``` or ```[[/attack]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="attack" data-formula="+8" data-activity-uuid="...uuid...">
+ *   <i class="fa-solid fa-dice-d20" inert"></i> +8
+ * </a>
+ * ```
  */
 async function enrichAttack(config, label, options) {
+  if ( config.activity && config.formula ) {
+    console.warn(`Activity ID and formula found while enriching ${config._input}, only one is supported.`);
+    return null;
+  }
+
   const formulaParts = [];
   if ( config.formula ) formulaParts.push(config.formula);
-  for ( const value of config.values ) formulaParts.push(value);
+  for ( const value of config.values ) {
+    if ( value in CONFIG.DND5E.attackModes ) config.attackMode = value;
```

Comment:
Maybe just `mode`? This is the `attack` enricher after all so writing `[[/attack attackMode=foo]]` feels a bit redundant. I don't feel strongly though, given that the `attackMode` key can be entirely omitted, which is probably a more idiomatic.

Reply:
I'd say yes if it wasn't also used in the damage enricher, where `mode` is a bit more ambiguous.